instrumentation/mybatis/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/mybatis/MybatisSingletons.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.mybatis;
+
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.instrumentation.api.instrumenter.Instrumenter;
+import io.opentelemetry.instrumentation.api.instrumenter.SpanKindExtractor;
+import io.opentelemetry.instrumentation.api.instrumenter.SpanNameExtractor;
+
+public final class MybatisSingletons {
+  private static final String INSTRUMENTATION_NAME = "io.opentelemetry.mybatis";
```

Comment:
by convention we add the minimal supported version to the instrumentation name, so here it should be `io.opentelemetry.mybatis-3.2`. As instrumentation name here has to match the module name in the code you should also rename mybatis -> mybatis-3.2

Reply:
> by convention we add the minimal supported version to the instrumentation name, so here it should be `io.opentelemetry.mybatis-3.2`. As instrumentation name here has to match the module name in the code you should also rename mybatis -> mybatis-3.2

I adjusted it.